lib/ttfunk/subset/windows_1252.rb

```diff
@@ -11,41 +10,46 @@ def initialize(original)
         @subset = Array.new(256)
       end
 
+      def code_page
+        'cp1252'
+      end
+
       def to_unicode_map
-        Encoding::Windows1252::TO_UNICODE
+        CodePages[code_page].unicode_mapping
```

Comment:
After all the changes this class is different to `MacRoman` only by the `code_page` method. Moreover, this one still encodes into `mac_roman` cmap table. Do we need it?

Reply:
Yeah, I noticed that as well. In order to not break backwards-compatibility, I'll keep these two classes but refactor into a base class that can accept any encoding/code page.